src/components/absorber/module.cairo

```diff
@@ -154,6 +159,22 @@ mod Absorber {
                 );
         }
     }
+
+    #[generate_trait]
+    impl PrivateImpl of PrivateTrait {
```

Comment:
This could be an internal function, without the need of generate_trait and ContractState